packages/enrichPrescriptions/src/statusUpdates.ts

```diff
@@ -109,20 +120,36 @@ export function applyStatusUpdates(searchsetBundle: Bundle, statusUpdates: Statu
 
     logger.info(`Applying updates for prescription ${prescriptionID}.`)
 
-    const prescriptionUpdate = statusUpdates.prescriptions.filter(p => p.prescriptionID === prescriptionID)[0]
+    const prescriptionUpdate = statusUpdates.prescriptions.filter((p) => p.prescriptionID === prescriptionID)[0]
     if (!prescriptionUpdate || !prescriptionUpdate.onboarded) {
       logger.info(`Supplier of prescription ${prescriptionID} not onboarded. Applying default updates.`)
-      medicationRequests?.forEach(medicationRequest =>
+      medicationRequests?.forEach((medicationRequest) => {
+        if (delayWithPharmacyStatus(medicationRequest)) {
+          logger.info(
+            `Delaying 'With Pharmacy' status for prescription ${prescriptionID} line item id ${medicationRequest.id}`
+          )
+          // If the prescription has been in "With Pharmacy" status for less than an hour,
```

Comment:
I think this log and comment are a bit misleading. Shouldn't they be '_..."With Pharmacy but Tracking not Supported" status..._'?

Plus, it's True at this point, so no 'if' required.

Reply:
e3e76eff7306ccb2f4025a4f509860d07431ae7f